digital_benchmark/accounts/views.py

```diff
@@ -51,9 +55,14 @@ def post(self, request):
                 return redirect(next)
             return redirect('/')
 
-        return render(request,'signup.html',{'form':form})
+        return render(request, 'signup.html', {'form': form})
+
 
 class LogoutView(View):
     def get(self, request):
         logout(request)
-        return redirect('/')
\ No newline at end of file
+        return redirect('/')
+
+
+class RegisterUserView(generics.CreateAPIView):
+    serializer_class = UserSerializer
```

Comment:
@azsidhu see what happens with password field when user is created through this endpoint. Does password gets saved as hashed value or plain value.

Reply:
It's being saved as hashed value, because I'm overriding create method in UserSerializer.